README.md

```diff
@@ -187,6 +187,37 @@ result = my_zosmf.get_info()
 ```
 The result will be a JSON object containing z/OSMF information
 
+## Local project setup (Windows)
+Enlisted below are the project setup instructions.
+`Note` - Usage of `python3.10` or older is recommended.
+
+
+0. Pre Step - Switch to branch `next`
```

Comment:
Please start the list of steps as 1-indexed to maintain consistency with our other docs 🙂 "Pre step" can be omitted.